ibis/backends/mssql/__init__.py

```diff
@@ -133,6 +133,9 @@ def do_connect(
     def get_schema(
         self, name: str, *, catalog: str | None = None, database: str | None = None
     ) -> sch.Schema:
+        if name.startswith("ibis_cache_"):
```

Comment:
Probably should have a `TODO` here to have a slightly more robust solution to the problem of cached table lookups. I can definitely see non-internal uses of a table with this prefix.